processor/tailsamplingprocessor/config_test.go

```diff
@@ -114,6 +116,17 @@ func TestLoadConfig(t *testing.T) {
 						BooleanAttributeCfg: BooleanAttributeCfg{Key: "key4", Value: true},
 					},
 				},
+				{
+					sharedPolicyCfg: sharedPolicyCfg{
+						Name: "test-policy-11",
+						Type: OTTLCondition,
+						OTTLConditionCfg: OTTLConditionCfg{
+							ErrorMode:           ottl.IgnoreError,
+							SpanConditions:      []string{"attributes[\"test_attr_key_1\"] == \"test_attr_val_1\"", "attributes[\"test_attr_key_2\"] != \"test_attr_val_1\""},
```

Comment:
Out of curiosity: would single quotes work to wrap the attribute names (such as `test_attr_key_1`)?

Reply:
No, OTTL requires double quotes `""`.  In situations like these you could create the strings using back ticks ` instead of ".